<launch>
        <!-- global params -->
        <param name="/use_sim_time" value="false"/>
        <rosparam file="$(find hole_tracker)/config/main.yaml" command="load" ns="hole_tracker"/>

        <!-- launch node -->
        <node name="tracker" pkg="hole_tracker" type="node_tracker.py" output="screen">

                <!-- input topics -->
                <remap from="input_points" to="/tracker/detector/points"/>
                <remap from="input_img" to="/quail/wrist_cam/image_raw/compressed"/>
                <remap from="input_odom" to="/quail/odometry"/>
                <remap from="input_normal" to="/quail/normal_estimation/pose"/>
                <remap from="input_uavstate" to="/quail/controller_node/uav_state"/>
                <remap from="input_depthmap" to="/tracker/depthmap"/>

                <!-- output topics -->
                <remap from="output_img" to="/tracker/img/compressed"/> <!-- /compressed subtopic for rviz! -->
                <remap from="output_estim" to="/tracker/estimate"/>

        </node>
</launch>
